lxd/lifecycle/lifecycle.go

```diff
@@ -0,0 +1,27 @@
+package lifecycle
+
+import "fmt"
+
+type InstanceAction string
+
+const (
+	InstanceCreated   = InstanceAction("created")
+	InstanceStarted   = InstanceAction("started")
+	InstanceStopped   = InstanceAction("stopped")
+	InstanceShutdown  = InstanceAction("shutdown")
+	InstanceRestarted = InstanceAction("restarted")
+	InstancePaused    = InstanceAction("paused")
+	InstanceResumed   = InstanceAction("resumed")
+	InstanceRestored  = InstanceAction("restored")
+	InstanceDeleted   = InstanceAction("deleted")
+	InstanceRenamed   = InstanceAction("renamed")
+	InstanceUpdated   = InstanceAction("updated")
```

Comment:
All of the `network-*` ones should also be added here I think.

Reply:
You're also missing all of the `instance-backup` ones (created, renamed, deleted).